hpedockerplugin/cmd/cmd_setquota.py

```diff
@@ -56,6 +57,7 @@ def _update_share_metadata(self, quota_id, add=True):
             share['status'] = 'AVAILABLE'
         elif 'quota_id' in share:
             share.pop('quota_id')
+            share['status'] = 'FAILED'
```

Comment:
@imran-ansari  @wdurairaj  we are setting here status of share as failed which is correct thing to do.
Hence this share should get deleted by user. 
Delete flow should remove not only etcd entry but should also remove backend share present on 3PAR.
Please  verify if current flow takes care of this.

Reply:
@imran
along with deleting etcd record we also need to check if share with that name exist on 3PAR and delete it as well. (in delete share flow)

Reply:
@nilangekarss - setting of status to 'FAILED' is happening in unexecute i.e. rollback flow... we are setting it to FAILED just for the sake of completeness... Removal of share/file-store/fpg happens in the unexecute of cmd_createshare which is the next command on the undo_cmds stack.

Reply:
@imran-ansari Okay Got it.

Reply:
@imran-ansari under undo steps, we are deleting the etcd record for the share but we are not returning a failed response for docker engine.
I believe either we should send a failed response to docker engine  or etcd record should be kept as it is so that user can see share in failed state and can delete share with docker command.

need your thoughts on this.

Reply:
> 
> 
> @imran
> along with deleting etcd record we also need to check if share with that name exist on 3PAR and delete it as well. (in delete share flow)

Hi guy's, your quoting the wrong imran @sandesh-desai @nilangekarss